src/test/java/com/google/gcp_variant_transforms/library/HeaderIteratorTest.java

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import static com.google.common.truth.Truth.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gcp_variant_transforms.library.HeaderIterator;
+import org.junit.*;
+
+/**
+ * Units tests for HeaderIterator.java
+ */
+public class HeaderIteratorTest {
+
+  public HeaderIterator createHeaderIterator(int numHeaderLines){
+    ImmutableList.Builder<String> headerLinesBuilder = new ImmutableList.Builder<String>();
+    HeaderIterator headerIterator;
+    for (int i = 0; i < numHeaderLines; i++) {
+      headerLinesBuilder.add("##sampleHeaderLine=" + i);
+    }
+    headerIterator = new HeaderIterator(headerLinesBuilder.build());
+    return headerIterator;
+  }
+
+  @Test
+  public void testHeaderLinesHasNext_whenCheckingBoolean_thenTrue() {
+    // Arrange
```

Comment:
I don't think we actually need these comments, as long as we break them into correct blocks (//Arrange, //Act, //Assert). Lets remove them.

Reply:
Apologies, I misunderstood your last comment on these. They're taken out.
resolved